Common subdirectories: audible-activator.orig/.git and audible-activator.new/.git
diff -U3 audible-activator.orig/audible-activator.py audible-activator.new/audible-activator.py
--- audible-activator.orig/audible-activator.py	2017-05-25 08:24:36.000000000 +0200
+++ audible-activator.new/audible-activator.py	2017-05-25 13:31:00.000000000 +0200
@@ -57,7 +57,7 @@
         if sys.platform == 'win32':
             chromedriver_path = "chromedriver.exe"
         else:
-            chromedriver_path = "./chromedriver"
+            chromedriver_path = "chromedriver"
 
         driver = webdriver.Chrome(chrome_options=opts,
                                   executable_path=chromedriver_path)
Common subdirectories: audible-activator.orig/unused and audible-activator.new/unused
